contribute/design/1.x/csi/20190606-csi-volume-provisioning.md

```diff
@@ -272,65 +272,36 @@ smooth volume create & delete operations.
 
 #### CStorVolumeConfigClass -- new custom resource
 This is a new Kubernetes custom resource that holds config information to be
-used by cstor volume claim controller while provisioning cstor volumes.
+used by cstor volume claim controller while provisioning cstor volumes. For each 
+volume corresponding CstorVolumeConfig is created.
```

Comment:
are we going away with existing CStorVolume CR with addition of CStorVolumeConfig? if not, can these both merged?